n/nbconvert/build_info.json

```diff
@@ -8,7 +8,7 @@
     "package_dir": "n/nbconvert/",
     "docker_build": false,
     
-    "v7.7.3": {
+    "v7.12.0": {
```

Comment:
you can append new version string /pattern or add new entry. need to keep old version supported

Reply:
Okay

Reply:
Made required changes.